cmd/rpcfuzz/rpcfuzz.go

```diff
@@ -703,6 +730,21 @@ func setupTests(ctx context.Context, rpcClient *rpc.Client) {
 		Validator: ValidateRegexString(`^0x$`),
 		Flags:     FlagStrictValidation,
 	})
+	allTests = append(allTests, &RPCTestGeneric{
+		Name:   "RPCTestEthCallRevertMessage",
+		Method: "eth_call",
+		Args:   []interface{}{&RPCTestTransactionArgs{To: ltEthAddress, Data: "0xa26388bb"}, "latest"},
+		Flags:  FlagErrorValidation,
+		Validator: func() func(result interface{}) error {
+			cops := new(bind.CallOpts)
+			ltRevertMsg, ltCallErr := ltContract.RevertErrorMessage(cops)
```

Comment:
I think it would be fine to hard code a revert message that we're using for testing purposes.

Reply:
True, but I like how I don't have to maintain a variable outside of the solidity file